rpc/wrtc_call_queue_mongodb.go

```diff
@@ -597,15 +602,21 @@ func (queue *mongoDBWebRTCCallQueue) processNextSubscriptionEvent(next mongoutil
 			if _, ok := queue.csTrackingHosts[callResp.Host]; !ok {
 				// no one connected to this operator is currently subscribed to insert
 				// events for this host; skip
+				// we do this because each server is listening to an event and each host only lives on one server
+				// it could be on another server
 				return
```

Comment:
would it be worth adding a `debug` log here or would it be so insanely noisy that it's not useful?

Reply:
i think it would be quite noisy and not useful

Reply:
thanks!